yaksh/templates/yaksh/grade_user.html

```diff
@@ -222,15 +219,16 @@ <h5>No Assignment submitted by {{ data.user.get_full_name.title }}</h5>
     {% endif %}
     {% else %}
         {% for ans in answers %}
-        {% if ans.answer.correct %}
+            <!-- {% if ans.answer.correct %}
                 <div class="card ">
-                    <div class="card-heading alert-success">Correct answer:
-            {% else %}
+                   <div class="card-heading alert-success">Correct answer:</div></div> -->
+            <!-- {% else %} --><!-- 
                 <div class="card ">
-                    <div class="card-heading-heading alert-danger">Error:
-            {% endif %}
-            {% with ans.error_list as err %}
-            {% for error in err %}
+                    <div class="card-heading-heading alert-danger">Error: -->
+
+            <!-- {% endif %} -->
+            <!-- {% with ans.error_list as err %} -->
+            <!-- {% for error in err %}
```

Comment:
Why is this commented? If it is not required then remove otherwise keep it as it is. Check the working before you remove any content.